infra/recipes/azure-sandbox/config/providers.tf

```diff
@@ -0,0 +1,10 @@
+provider "azurerm" {
+  features {}
+}
+
+provider "kubernetes" {
+  host                   = azurerm_kubernetes_cluster.sandbox.kube_config.0.host
```

Comment:
Where these variables are used? What is the significance of these variables in output.tf. Can't they be used directly?

Reply:
I believe that the output variables are to expose values to the command line and other programs. Right now, the variables in output.tf are not used but are helpful for debugging/gaining understanding of the services.